cogs/events/events.py

```diff
@@ -618,7 +632,12 @@ def setup(bot):
                 vag_words,
                 False):
             await bot.add_reaction(message, '😞')
-
+        # NEW (MM): check for punny words and respond
+        elif list(set(keywords) & set(mset))
```

Comment:
Needs colon after `elif`, but if we change to dict that we'll need to do instead:

```python
trigger = set(triggers.keys()).union(message_split)
elif len(trigger) != 0:
```